init/init.c

```diff
@@ -582,7 +583,13 @@ static int config_parse_file(char ***argv, char **workdir)
 	int fd;
 	int i;
 
-	fd = open(CONFIG_FILE_PATH, O_RDONLY);
+	config_file = getenv("KRUN_CONFIG");
+	if (config_file) {
+		fd = open(config_file, O_RDONLY);
+	} else {
+		fd = open(CONFIG_FILE_PATH, O_RDONLY);
+	}
+
```

Comment:
A bit cleaner, maybe:

```
	config_file = getenv("KRUN_CONFIG");
	if (!config_file) {
                config_file = CONFIG_FILE_PATH;
       }
	fd = open(config_file, O_RDONLY);
```

Reply:
Or if you don't care about MSVC, `open(getenv("KRUN_CONFIG") ?: CONFIG_FILE_PATH, O_RDONLY)` is a one-liner with the elvis operator :man_dancing: